test/e2e/app/main.go

```diff
@@ -235,15 +250,16 @@ func startMaverick(cfg *Config) error {
 		misbehaviors[height] = mcs.MisbehaviorList[misbehaviorString]
 	}
 
+	// TODO: What is a maverick node?
 	n, err := maverick.NewNode(tmcfg,
 		maverick.LoadOrGenFilePV(tmcfg.PrivValidatorKeyFile(), tmcfg.PrivValidatorStateFile()),
 		nodeKey,
 		proxy.NewLocalClientCreator(app),
 		maverick.DefaultGenesisDocProviderFunc(tmcfg),
 		maverick.DefaultDBProvider,
 		maverick.DefaultMetricsProvider(tmcfg.Instrumentation),
+		dashCoreRpcClient,
 		logger,
-		misbehaviors,
```

Comment:
This can't be removed

Reply:
But this method doesn't accept anything related to misbehaviors, there's just no such argument in this function